slices/dotnet-sdk-aot-9.0.yaml

```diff
@@ -0,0 +1,13 @@
+package: dotnet-sdk-aot-9.0
+
+essential:
+  - dotnet-sdk-aot-9.0_copyright
+
+slices:
+  libs:
+    contents:
```

Comment:
This needs whatever gets decided on the [25.04 slice](https://github.com/canonical/chisel-releases/pull/561#discussion_r2395026001)